src/wranglerAction.ts

```diff
@@ -241,26 +241,6 @@ function getEnvVar(envVar: string) {
 	return value;
 }
 
-async function legacyUploadSecrets(
```

Comment:
Should we keep the legacy upload behavior for people on older versions of wrangler?

Reply:
Ohh, `"wrangler", "secret:bulk"` is being deprecated, and `"wrangler", "secret", "put"` is the current "legacy" way of doing things

Reply:
The legacy behavior _is_ the current behavior :) The "new" behavior was the now-deprecated secret:bulk command.

All this PR does is replace the "non-legacy" behavior with the code from this function